From: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Subject: override `config/chroot`
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Description:
This patch overrides the `config/chroot` file created by `./build.sh`

--- a/config/chroot
+++ b/config/chroot
@@ -14,7 +14,7 @@

 # $LB_KEYRING_PACKAGES: set keyring packages
 # (Default: empty)
-LB_KEYRING_PACKAGES="kali-archive-keyring"
+LB_KEYRING_PACKAGES="ca-certificates kali-archive-keyring"

 # $LB_LINUX_FLAVOURS: set kernel flavour to use
 # This is kept for backwards compatibility
